zboxcore/sdk/blobber_operations.go

```diff
@@ -160,6 +163,24 @@ func UpdateAllocation(
 	updateAllocationRequest["owner_signing_public_key"] = ownerSigninPublicKey
 	updateAllocationRequest["file_options_changed"], updateAllocationRequest["file_options"] = calculateAllocationFileOptions(alloc.FileOptions, fileOptionsParams)
 
+	if ticket != "" {
+
+		type Ticket struct {
```

Comment:
extract type

Reply:
It is part of signature so I think we can get it in input from user.